src/CMakeLists.txt

```diff
@@ -26,6 +26,9 @@ set_target_properties(mc_udp_data PROPERTIES COMPILE_FLAGS "-DMC_UDP_DATA_EXPORT
 target_include_directories(mc_udp_data PUBLIC
   $<BUILD_INTERFACE:${PROJECT_SOURCE_DIR}/include>
   $<INSTALL_INTERFACE:include>)
+if(BUILD_LEGACY_OPENRTM_SERVER)
+  target_compile_options(mc_udp_data PUBLIC -std=c++0x)
```

Comment:
Is this really needed? I think `set(CMAKE_CXX_STANDARD 11)` handles this correctly and a CMake version that has `target_compile_options` should know to set `-std=c++0x` on older compilers (but I might be too optimistic)

Reply:
Sadly it seems to be needed. Without it on the robot you get compilation pragmas suggesting to turn on `c++0x` and it seems that no C++11 features are supported (at least `auto` was not supported).